recipes/vmd/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+mesa-libGL-devel
+libXinerama-devel
+libXi-devel
```

Comment:
Is it possible to use either conda-forge packages or CDTs for these?

Reply:
It cannot find `libGL.so.1` if I use ` {{ cdt('mesa-libGL-devel') }}`. [Here is the log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=12135). How can I solve it? 

> /home/conda/feedstock_root/build_artifacts/vmd_1549891315266/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/vmd_LINUXAMD64: error while loading shared libraries: libGL.so.1: cannot open shared object file: No such file or directory